program/src/instruction.rs

```diff
@@ -41,6 +41,15 @@ pub enum PaladinGovernanceInstruction {
     /// 0. `[s]` Paladin stake authority account.
     /// 1. `[w]` Proposal account.
     CancelProposal,
+    /// Finalize a draft governance proposal and begin voting.
+    ///
+    /// Authority account provided must be the proposal creator.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    /// 0. `[s]` Paladin stake authority account.
+    /// 1. `[w]` Proposal account.
+    BeginVoting,
```

Comment:
I can't comment on line `71`, but the first line of comments on the `SwitchVote` needs updating (it is showing the same as the `Vote` instruction).

Reply:
Great thanks! I'll probably address this in another PR.